pkg/cnab/config-adapter/adapter.go

```diff
@@ -2,7 +2,7 @@ package configadapter
 
 import (
 	"fmt"
-	"path/filepath"
+	unix_path "path"
```

Comment:
lol not sure we need to name the package unix_path 😉 Maybe just put a comment where we use it that we have to use unix paths because the path is used inside the bundle?